ui/components/app/create-new-vault/create-new-vault.js

```diff
@@ -144,9 +143,9 @@ export default function CreateNewVault({
       <Button
         data-testid="create-new-vault-submit-button"
         className="create-new-vault__submit-button"
-        type="primary"
-        submit
+        variant={BUTTON_VARIANT.PRIMARY}
         disabled={!isValid}
+        type="submit"
```

Comment:
Apologies @PrgrmrHarshShukla, after second review I think we do want to retain that submit attribute I've updated it here